docker/include/bin/start_faux

```diff
@@ -132,9 +132,12 @@ elif [ -n "${options[bacnetfail]}" ]; then
          FauxDeviceEngine.EntryPoint $local_ip $broadcast_ip "Faux-Device-Fail.json"  &
 fi
 
-if [ -n "${options[ntp_client]}" ]; then
+if [ -n "${options[ntpv4fail]}" ]; then
     echo Starting ntp client.
     java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar "time.google.com" "123" "3" &
+elif [ -n "${options[ntpv4pass]}" ]; then
```

Comment:
as per other comment, can you make the two cases more similar -- e.g. both should have the same number of arguments. I don't see any reason why anything should be optional here.